src/components/BottomNavbar.jsx

```diff
@@ -38,12 +41,18 @@ export function BottomNavbar() {
       setpostfix(Constants.readMore);
     }
     setisCollapsed(!isCollapsed);
+
+    //only for mobile phones (handling the overflowed chat arrow element `gooey-message:after`)
+    if (window.outerWidth <= 600) {
+      gooeyMessage.current.style.overflow =
+        gooeyMessage.current.style.overflow === 'auto' ? 'visible' : 'auto';
```

Comment:
I think you can accomplish this with css.

https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

Reply:
@pollend Thanks for the review :) 👍


I have used the media queries, but this style/dom modification is based on a button click.

I.e. The styling needs to be changed on button click (read more button)

Intention - to set the overflow to auto if the message is longer (on button click) 

Again set the overflow to visible after clicking on see less.

Reason - We've used :after property for the message bubble `arrow` in gooey message. So if we set overflow:auto (which makes the message scrollable) then :after gets invisible (due to overflow behaviour)

Reply:
you can append a class conditionally with react by passing it to a prop. have a boolean flag and some condition for the class to be used.

Reply:
Ohk @pollend will give it a try :) thanks btw :)

Reply:
@pollend Done. Kindly have a look.